app/javascript/tests/enrollment/steps/ExposureInformation.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import ExposureInformation from '../../../components/enrollment/steps/ExposureInformation';
+import { blankMockPatient, mockPatient2 } from '../../mocks/mockPatients';
+import { mockJurisdiction1, mockJurisdictionPaths } from '../../mocks/mockJurisdiction';
+import { mockLaboratory1 } from '../../mocks/mockLaboratories';
+
+const newEnrollmentState = {
+  isolation: false,
+  patient: blankMockPatient,
+  propagatedFields: {},
+};
+
+const requiredStrings = ['LAST DATE OF EXPOSURE', 'EXPOSURE LOCATION', 'EXPOSURE COUNTRY', 'CONTINUOUS EXPOSURE', 'EXPOSURE RISK FACTORS (USE COMMAS TO SEPARATE MULTIPLE SPECIFIED VALUES)', 'NOTES'];
+
+describe('ExposureInformation', () => {
+  it('Properly renders all main components', () => {
+    const wrapper = mount(<ExposureInformation goto={() => {}} next={() => {}} setEnrollmentState={() => {}} currentState={newEnrollmentState} patient={blankMockPatient} has_dependents={false} jurisdiction_paths={mockJurisdictionPaths} assigned_users={[]} selected_jurisdiction={mockJurisdiction1} first_positive_lab={mockLaboratory1} edit_mode={false} authenticity_token={'123'} />);
+    requiredStrings.forEach(requiredString => {
+      expect(wrapper.text().includes(requiredString)).toBe(true);
+    });
+  });
+
+  it('Properly shows the `Next` button', () => {
```

Comment:
you could probably add this as a part of properly renders all main components, same with the previous button, and then just have one test that makes sure the previous button is properly hidden